deployment/tensorrt/main.cpp

```diff
@@ -304,7 +357,14 @@ std::vector<Detection> YOLOv5Detector::detect(cv::Mat& image) {
   int32_t input_h = engine->getBindingDimensions(0).d[2];
   int32_t input_w = engine->getBindingDimensions(0).d[3];
   cv::Mat tmp;
-  cv::resize(image, tmp, cv::Size(input_w, input_h));
+  /* Fixed shape, need to set h/w scale */
+  float scale = letter_box(
```

Comment:
Could we rename this function to `letterbox` for the sake of consistency with the python.